projects/worker-basic/package.json

```diff
@@ -0,0 +1,35 @@
+{
```

Comment:
Same comments as for the other `package.json`: https://github.com/adobe/asset-compute-example-workers/pull/1/files?file-filters%5B%5D=.jpg&file-filters%5B%5D=.js&file-filters%5B%5D=.json&file-filters%5B%5D=.txt&file-filters%5B%5D=.yml&file-filters%5B%5D=No+extension#diff-8526336fa800f96e8fffec85fd015345